gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -258,15 +264,23 @@ def eink_st_sonderausgaben_ab_2012(
         12 * betreuungskost_m,
         eink_st_abzuege_params["kinderbetreuungskosten_abz_maximum"],
     )
+    sonderausgaben_betreuung = (
+        anz_kinder_tu
+        * abziehbare_betreuungskosten
+        * eink_st_abzuege_params["kinderbetreuungskosten_abz_anteil"]
+    ) / anz_erwachsene_tu
+
+    if gemeinsam_veranlagt_tu:
+        pauschale = eink_st_abzuege_params["sonderausgabenpauschbetrag"]["couple"]
+    else:
+        pauschale = eink_st_abzuege_params["sonderausgabenpauschbetrag"]["single"]
 
     if kind:
         out = 0.0
+    elif sonderausgaben_betreuung > pauschale:
+        out = sonderausgaben_betreuung
```

Comment:
We should add a test case for which `sonderausgaben_betreuung > pauschale` equals to true. This seems to be never the case.

Reply:
I added two test cases, which I had to construct myself since I, again, couldn't find an entire calculation example online. So I referred to one previous example from one of the online calculators. The second test case includes one child which is above the age threshold of 14.

Reply:
Looks great!